From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Fri, 8 Dec 2023 10:19:57 +0100
Subject: osk-manager: Sync initial state
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Fixes 43d801d9da2dfe2b90f6b38068c3517d3e658d39

Tested-by: Jarrah Gosbell <debian@undef.tools>
Signed-off-by: Guido Günther <agx@sigxcpu.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1332>
(cherry picked from commit f8b1aded6ab56b2200f068ab6f9fc11a421ee4ec)

Closes: #1057766
---
 src/osk-manager.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/osk-manager.c b/src/osk-manager.c
index 9c0573a..73ec150 100644
--- a/src/osk-manager.c
+++ b/src/osk-manager.c
@@ -127,7 +127,7 @@ dbus_name_owner_changed_cb (PhoshOskManager *self, gpointer data)
 
 
 static void
-on_screen_keyboard_enabled_changed (PhoshOskManager *self, GParamSpec *pspec, gpointer unused)
+on_screen_keyboard_enabled_changed (PhoshOskManager *self)
 {
   gboolean enabled;
 
@@ -281,6 +281,7 @@ phosh_osk_manager_init (PhoshOskManager *self)
                             "changed::screen-keyboard-enabled",
                             G_CALLBACK (on_screen_keyboard_enabled_changed),
                             self);
+  on_screen_keyboard_enabled_changed (self);
 }
 
 
